apps/system/js/advanced_telemetry.js

```diff
@@ -575,16 +575,26 @@
 
     // Pack the Addon Histograms.
     for (var addon in nameAddonHist) {
-      for (var hist in nameAddonHist[addon]) {
-        nameAddonHist[addon][hist] =
-          this.packHistogram(nameAddonHist[addon][hist]);
+      // Discard any Histograms with no data.
+      if (Object.keys(nameAddonHist[addon]).length === 0) {
+        delete nameAddonHist[addon];
+      } else {
```

Comment:
I'm channeling for Jan... the code is cleaner if it returns when the histogram is empty and then doesn't need the 'else'